### ISSUE TITLE
Fix the case that shader might compile error in osx.

### ISSUE THREAD

**Issue by andy840119** [karaoke-dev/osu-framework-font#82]
Error:
```
osu.Framework.Graphics.Shaders.Shader+PartCompilationFailedException: A osu.Framework.Graphics.Shaders.ShaderPart failed to compile: sh_CRT.fs:
ERROR: 0:58: 'f' : syntax error: syntax error
   at osu.Framework.Graphics.Shaders.ShaderPart.Compile()
   at osu.Framework.Graphics.Shaders.Shader.CompileInternal()
   at osu.Framework.Graphics.Shaders.Shader.compile()
   at osu.Framework.Threading.ScheduledDelegate.RunTaskInternal()
   at osu.Framework.Threading.ScheduledDelegate.RunTask()
   at osu.Framework.Graphics.OpenGL.GLWrapper.Reset(Vector2 size)
   at osu.Framework.Platform.GameHost.DrawFrame()
   at osu.Framework.Threading.GameThread.processFrame()
--- End of stack trace from previous location ---
   at osu.Framework.Platform.GameHost.<>c__DisplayClass120_0.<abortExecutionFromException>b__0()
   at osu.Framework.Threading.ScheduledDelegate.RunTaskInternal()
   at osu.Framework.Threading.Scheduler.Update()
   at osu.Framework.Threading.GameThread.processFrame()
   at osu.Framework.Threading.GameThread.RunSingleFrame()
   at osu.Framework.Platform.ThreadRunner.RunMainLoop()
   at osu.Framework.Platform.GameHost.windowUpdate()
   at osu.Framework.Platform.SDL2DesktopWindow.Run()
   at osu.Framework.Platform.GameHost.Run(Game game)
   at osu.Framework.Font.Tests.Program.Main(String[] args) in /Volumes/Data/Github/karaoke-dev/osu-framework-font/osu.Framework.Font.Tests/Program.cs:line 21
```

It might by caused by "compile error", not "file not found"

**Comment by andy840119** [comment]
Maybe should move the test case into osu-framework then run again.

**Comment by andy840119** [comment]
It will cause same issue in osu-framework also.
Need to check the reason.

**Comment by andy840119** [comment]
update:
it might caused by osx cannot compile `sh_CRT.fs` or `sh_Outline.fs`
not really why.

**Comment by andy840119** [comment]
https://www.coder.work/article/749628
https://stackoverflow.com/questions/28435438/how-to-write-const-array-in-glsl-es
not really sure the reason is osx does not support ES 3.0?

**Comment by andy840119** [comment]
https://www.youtube.com/watch?v=lRya2Pclpy8
*When i figure out that osx does not support OpenGL ES 2.0

**Comment by andy840119** [comment]
Guess the laziest way is just to place it and wait until it is fixed.
support osx might not be the most important one.

**Comment by andy840119** [comment]
Fixed in the #140 and #147
Need the damn osx for developing, so there's no other choice.
